.github/workflows/advanced.yml

```diff
@@ -93,10 +93,10 @@ jobs:
         with:
           path: ${{ github.workspace }}
 
-      - name: Get workflow SHA
-        id: workflow-sha
+      - name: Get scripts SHA
+        id: scripts-sha
         run: |
-          echo sha=$(git log -1 --pretty=format:"%H" .github/workflows/ .github/scripts) >> "$GITHUB_OUTPUT"
+          echo sha=$(git log -1 --pretty=format:"%H" .github/workflows/advanced.yml .github/scripts/binutils .github/scripts/toolchain) >> "$GITHUB_OUTPUT"
```

Comment:
binutils and toolchain I would expect in in binutils anf gcc hashes

Reply:
I am not sure what are you suggesting? The `toolchain` directory contains scripts for building both MinGW, GCC, and GCC librarires.

Reply:
to calculate hash for workflow it uses binutils and toolchain folders with scripts, which I would exclude from workflow (scripts) sha
Instead I would use scripts folders for binutils and gcc hashes

Reply:
maybe scripts sha needs to be renamed to workflow sha

Reply:
OK, now I understand.